src/Adapter/Product/QueryHandler/GetProductForEditingHandler.php

```diff
@@ -58,13 +65,21 @@ final class GetProductForEditingHandler extends AbstractProductHandler implement
      */
     private $numberExtractor;
 
+    /**
+     * @var StockAvailableRepository
+     */
+    private $stockAvailableRepository;
+
     /**
      * @param NumberExtractor $numberExtractor
+     * @param StockAvailableRepository $stockAvailableRepository
      */
     public function __construct(
-        NumberExtractor $numberExtractor
+        NumberExtractor $numberExtractor,
+        StockAvailableRepository $stockAvailableRepository
```

Comment:
BC Break

Reply:
ProductForEditingHandler not released yet, no bc break :stuck_out_tongue:

Reply:
Correct ;)